services/provider/consumerstatus/status.go

```diff
@@ -0,0 +1,9 @@
+package consumerstatus
+
+type ConsumerStatus interface {
```

Comment:
This is not an interface for the consumer status it is an interface to the storage client status. 
So maybe just `StorageClientStatus`?

Reply:
done.